prow/cmd/deck/main_test.go

```diff
@@ -291,7 +291,7 @@ func TestRerun(t *testing.T) {
 			allowAnyone:         false,
 			rerunCreatesJob:     true,
 			shouldCreateProwJob: true,
-			httpCode:            http.StatusFound,
+			httpCode:            http.StatusOK,
```

Comment:
IMO this might just be easier as 200 or whatever.

Reply:
I prefer it this way to stay consistent with other tests, and I think `StatusOK` is more informative for someone who isn't that familiar with the codes (me)